app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/DummyCategorizedPictureTestActivity.kt

```diff
@@ -0,0 +1,22 @@
+package com.github.HumanLearning2021.HumanLearningApp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.ImageView
+import com.github.HumanLearning2021.HumanLearningApp.Presenter.DummyUIPresenter
+import kotlinx.coroutines.launch
+import androidx.lifecycle.lifecycleScope
+
+class DummyCategorizedPictureTestActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_dummy_categorized_picture_test)
+
+        val dummyPresenter = DummyUIPresenter()
```

Comment:
pourquoi UI là dedans ?
```suggestion
        val dummyPresenter = DummyDatasetPresenter()
```

Reply:
The idea is that the Presenter is a big MVP block with more than just datasets concerns in it.